src/schema-generation/flex-search-filter-input-types/generator.ts

```diff
@@ -0,0 +1,276 @@
+        if (field.isList || !field.type.isEnumType) {
+            throw new Error(`Expected "${field.name}" to be a non-list enum`);
+        }
+        return ENUM_FILTER_FIELDS.map(name =>
+            new FlexSearchScalarOrEnumFieldFilterField(
+                field,
+                FILTER_OPERATORS[name],
+                name === INPUT_FIELD_EQUAL ? undefined : name, graphQLEnumType,
+                field.isFlexSearchIndexed ? field.language : undefined));
+    }
+
+    @memorize()
+    private generateListFieldFilterFields(field: Field, path?: ReadonlyArray<Field>): FlexSearchFilterField[] {
+        const pathParam = path ? path : [];
+        if (field.type instanceof ScalarType) {
+            return this.buildScalarFilterFields(field.type, field, pathParam);
+        } else if (field.type instanceof EnumType) {
+            return this.buildEnumFilterFields(field.type, field, pathParam);
+        } else {
+            const inputType = this.generate(field.type, true);
+            if (field.type.isEntityExtensionType) {
+                return [new FlexSearchEntityExtensionFilterField(field, inputType)];
+            } else {
+                return [new FlexSearchNestedObjectFilterField(field, inputType)];
+            }
+        }
+    }
+
+
+    private buildScalarFilterFields(type: ScalarType, field: Field, path?: ReadonlyArray<Field>): FlexSearchScalarOrEnumFilterField[] {
+        const filterFields = FLEX_SEARCH_FILTER_FIELDS_BY_TYPE[type.name] || [];
+
+        let scalarFields: FlexSearchScalarOrEnumFilterField[] = [];
+        if (field.isFlexSearchIndexed) {
+            scalarFields = scalarFields.concat(filterFields.map(name => new FlexSearchScalarOrEnumFilterField(field, FLEX_SEARCH_FILTER_OPERATORS[name], name, type.graphQLScalarType)));
+        }
+
+        if (field.language && field.isFlexSearchFulltextIndexed) {
+            scalarFields = scalarFields.concat(STRING_TEXT_ANALYZER_FILTER_FIELDS.map(name =>
+                new FlexSearchScalarOrEnumFilterField(
+                    field,
+                    this.getComplexFilterOperatorByName(name),
+                    name,
+                    type.graphQLScalarType,
+                    field.language)));
+        }
+
+        return scalarFields;
+
+    }
+
+    private buildEnumFilterFields(type: EnumType, field: Field, path?: ReadonlyArray<Field>) {
+        return ENUM_FILTER_FIELDS.map(name => {
+            return new FlexSearchScalarOrEnumFilterField(
+                field,
+                FLEX_SEARCH_FILTER_OPERATORS[name],
+                name,
+                this.enumTypeGenerator.generate(type));
+        });
+    }
+
+    private getValues(type: ObjectType, path: ReadonlyArray<Field>): ReadonlyArray<OrderByEnumValue> {
+        return flatMap(type.fields, field => this.getValuesForField(field, path));
+    }
+
+    private getValuesForField(field: Field, path: ReadonlyArray<Field>) {
```

Comment:
wait why is there orderBy code within filter input generators?

Reply:
it seems this is unused code, that was left over after a refactoring